### PR TITLE
manuSpecificTuya_1 as 0xE000 (57344) cluster

### PR DESCRIPTION
**Pull request by kirovilya**
girier module cluster
image

### CODE REVIEW

File: src/zcl/definition/cluster.ts [hunk 1]
```diff
@@ -4113,6 +4113,12 @@ const Cluster: {
         commands: {},
         commandsResponse: {},
     },
+    manuSpecificTuya_1: {
+        ID: 0xE000,
+        attributes: {},
+        commands: {},
+        commandsResponse: {},
```
Comment (Koenkk) [inline src/zcl/definition/cluster.ts:4120]: Since there are no attributes/commands/commandReponses how is this cluster used?

**Comment by kirovilya** [comment]
not used yet, but crashes if it cannot find such a cluster

**Comment by Koenkk** [comment]
> not used yet, but crashes if it cannot find such a cluster

what crashes?

**Comment by kirovilya** [comment]
sorry for bad picture (not my):
image

but it is not last herdsman...

**Comment by kirovilya** [comment]
image

**Comment by kirovilya** [comment]
this error appears 10 seconds after pairing a new device. apparently a message comes with this cluster - it falls like in the picture.
but if I add an empty cluster description, it will work fine.

I did not look in to the incoming message - this is happening for the user, not for me.

**Comment by Koenkk** [comment]
I believe this issue is already fixed in the latest herdsman due to https://github.com/Koenkk/zigbee-herdsman/commit/110b67a44bcefbafc3747ad0d8a0938eaa0dc40f , could you check that?

**Comment by kirovilya** [comment]
you are right. it works.

**Pull request closed without merging**